impl/src/main/java/org/jboss/cdi/tck/tests/full/interceptors/contract/interceptorLifeCycle/Bar.java

```diff
@@ -14,7 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.interceptors.tests.contract.lifecycleCallback;
+package org.jboss.cdi.tck.tests.full.interceptors.contract.interceptorLifeCycle;
```

Comment:
This is once again Git not being able to figure out what's going on. The other side of this is marked `Wai3Hesi`.